docs/user-guide/ze-profiles.md

```diff
@@ -67,3 +86,27 @@ Activate the SCS plug-in in Zowe Explorer.
 6. Restart VSCode.
 
 The credentials of your newly created or existing profiles are now stored securely.
+
+## Use Base Profile and Token with Existing Profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple Zowe services instantly. Base profiles enable you to authenticate via Zowe API Mediation Layer. You can use base profiles with more than one service profile. Any service profile with the same host and port as your base profile automatically uses your API ML token to log in.
+
+Before using the base profile functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v6.16 or higher installed.
+
+Connect your service profile with a base profile and token.
+
+**Follow these steps:**
+
+1. Open Zowe CLI and run the following command: `zowe auth login apiml`.
+
+2. Follow the instructions to complete the login.
+
+   A local base profile is created that contains your token. For more information about the process, see [Token Management](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#how-token-management-works).
+  
+3. Run Zowe Explorer and click the **+** icon.  
+
+4. Select the profile you use with your base profile with the token.
+
+   The profile appears in the tree and you can now use this profile to access z/OSMF via the API Mediation Layer.
+
+For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
```

Comment:
Good otherwise. We'll expand on this more in the near future, believe you me 👍

Reply:
I would maybe add one note that says something about, if you have an existing profile that you want to use with the new base profile, check that the service profile is using the same host & port as the base profile.

Reply:
@katelynienaber I moved this suggestion of yours to line 92. I didn't just cut it out :)